js/bybit.js

```diff
@@ -5117,54 +5616,65 @@ module.exports = class bybit extends Exchange {
          * @method
          * @name bybit#transfer
          * @description transfer currency internally between wallets on the same account
-         * @see https://bybit-exchange.github.io/docs/account_asset/#t-createinternaltransfer
+         * @see https://bybit-exchange.github.io/docs/account_asset/v3/#t-createinternaltransfer
```

Comment:
since we're using both methods here (v1, v3) I think we should keep both links

Reply:
ok